app/components/photo-tags/photo-tags.hbs

```diff
@@ -0,0 +1,41 @@
+<div class="position-relative photo-tags 
+  {{if showTags 'photo-tags--show'}}" 
+  {{on 'click' this.addTag}}
+  {{did-insert this.addCloseAddTagListener}}
+  {{will-destroy this.removeCloseAddTagListener}}
+>
+  {{yield}}
+
+  {{#if (gt @model.amountOfTags 0)}}
+    <button class="btn btn-info photo-tags-button" type="button" {{on 'click' this.toggleShowTags}}>
+      <FaIcon @icon='tag' />
+      {{ @model.amountOfTags }}
+    </button>
+  {{/if}}
+
+  {{#each @model.tags as |tag|}}
+    <div class="photo-tag" style={{ tag.tagStyle }}>
+      <LinkTo @route='users.user.photos' @model={{tag.taggedUser.id}}>
+        {{ tag.taggedUser.fullName }}
+      </LinkTo>
+      {{#if (can 'destroy photo-tag' tag)}}
+        <FaIcon @icon='xmark' {{ on 'click' (fn this.deleteTag tag) }} />
+      {{/if}}
+    </div>
+  {{/each}}
+
+  {{#if newTagStyle }}
+    <div class="photo-tag photo-tag--new" style={{ newTagStyle }}>
+      <PowerSelect
+        @options={{this.users}}
+        @onChange={{action 'storeTag'}}
+        @searchEnabled={{true}}
+        @searchField='fullName'
+        @registerAPI={{ action 'openUserSelect' }}
```

Comment:
https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/ the action helper is getting deprecated in favor of `@action` (which can be used in js files) paired with the helpers `{{on}}` or `{{fn}}` (which can be used in hbs files). I see you used it in other places, did you have a reason not to do so here? In these cases specifically, I suspect we may be able to do something like:
```suggestion
        @options={{this.users}}
        @onChange={{this.storeTag}}
        @searchEnabled={{true}}
        @searchField='fullName'
        @registerAPI={{this.openUserSelect}}
```
Though I will of course check whether this indeed works.
And I'm not entirely certain this is precisely what we should do (what about using `{{on 'change' this.storeTag}}` instead of my proposed `@onChange={{this.storeTag}}`? Would that work? )

Reply:
while applying this, I noticed that the powerselect includes users that have already been tagged. Then an error message is shown if you try to tag that user, which I like a lot. It makes sense because people may not know that the user is already tagged, and this informs them about it!
However, next step from the tagger's perspective in the case of a wrong location of the existing tag, is to correct its location. That's not directly possible with the current way permissions are setup, but they can I guess inform the user that's being tagged that the tag is incorrect, and then the tagged can remove the tag. In case of an inactive user, that is not really workable, and the person trying to tag needs to inform someone who has deletion rights (ICT for example). It's not an amazing situation to occur, but it may be rare, and I don't see an easy fix.